@types/ol-ext/overlay/Popup.d.ts

```diff
@@ -39,12 +39,13 @@ export declare type Template = {
  */
 
  export interface Options {
-    popupClass: string;
-    closeBox: boolean;
-    onclose: ((...params: any[]) => any) | undefined;
-    onshow: ((...params: any[]) => any) | undefined;
-    offsetBox: number | number[];
-    positionning: OverlayPositioning | string | undefined;
+    popupClass?: string;
+    anim?: boolean;
+    closeBox?: boolean;
+    onclose?: ((...params: any[]) => any) | undefined;
+    onshow?: ((...params: any[]) => any) | undefined;
+    offsetBox?: number | number[];
+    positioning?: OverlayPositioning | string | undefined;
```

Comment:
here you can also remove the undefined if you use the ?

Reply:
@Siedlerchr Okay, sure...
I felt that Options class `?` seems to be kind of default (also in `types-ol`), and actually `undefined` is too much, so I will remove `undefined` from here.
Thanks for advice!